openpathsampling/pathmover.py

```diff
@@ -384,6 +384,8 @@ def metropolis(cls, trials):
         accepted = True
         probability = 1.0
 
+        # TODO: This isn't right. `bias` should be associated with the 
+        # change; not with each individual sample. ~~~DWHS
```

Comment:
What would you call this then? The underlying idea was that since we accept samples we check, if all samples can be accepted and otherwise we multiply the `bias` which I think is something like the forward proposal probability.